core/src/main/resources/org/pentaho/platform/util/messages/messages.properties

```diff
@@ -71,4 +71,6 @@ LifecyclePhase.FAILED=Failed
 LifecyclePhase.FAILED_DESC=The work item execution has failed
 LifecyclePhase.RESTARTED=Restarted
 LifecyclePhase.RESTARTED_DESC=The work item execution has been restarted
+LifecyclePhase.REDIRECTED=Redirected
```

Comment:
I don't think the phases should be localized.  It makes it much more difficult to grep and for tools like kibana / ES to perform any metrics around a particular state.